manifests/base/runner-manager/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - name: runner-manager
+    newName: ghcr.io/scality/runner-manager
+    newTag: 0.6.3
```

Comment:
check if an image is available, could be one set with a git sha. An old release will not make your environment work.
```suggestion
    newTag: <git sha>
```

Reply:
```suggestion
```